workflows/argo-events/eventsources/k8s-secret-nautobot-token.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: argoproj.io/v1alpha1
+kind: EventSource
+metadata:
+  name: k8s-secret-nautobot-token
+  namespace: argo-events
```

Comment:
This should not be in the namespace and shouldn't be part of the argo-events deployment.